spark-rapids/spark-rapids.sh

```diff
@@ -428,7 +418,7 @@ function setup_gpu_yarn() {
 
   if [[ ${OS_NAME} == debian ]] || [[ ${OS_NAME} == ubuntu ]]; then
     export DEBIAN_FRONTEND=noninteractive
-    execute_with_retries "apt-get update"
+    execute_with_retries "apt-get --allow-releaseinfo-change update"
```

Comment:
FYI @jayadeep-jayaraman @viadea

Reply:
This is being addressed [here](https://github.com/GoogleCloudDataproc/initialization-actions/pull/1182)

Reply:
Can you please revert this change as we have a permanent fix. Once it is done I will merge the changes and will open a separate PR to ignore 2.2 ubuntu tests.

Reply:
Done!